update-url.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const glob = require("glob");
+
+const argIndex = process.argv.length-1;
+const env = process.argv[argIndex];
+
+const prod = '<script src="https://unpkg.com/thebelab@latest/lib/index.js"></script>'
+const staging = '<script src="LOCAL/index.js"></script>' // Need to change this
```

Comment:
We'll need to update this!

Reply:
Maybe the docs should always include a local build instead of sometimes referencing the CDN? Seems like each build of the docs should reference its own version of thebelab, since updates to the js could be incompatible with the examples in old versions of the docs.

Reply:
I might be misunderstanding, but are you suggesting we should default to the local build of Thebelab, or we have two separate builds?

My thought was that when there is a pull request, any changes to Thebelab would hopefully be reflected in the docs as well, so when Thebelab is deployed, new docs are also deployed in tandem.

I'm not sure how it works now, but that's how I assumed it might work :)

Reply:
I think that @minrk is suggesting that the docs should always be building from *master* of thebelab js, and not use the CDN for this.

Reply:
but, I can't tell whether @minrk is suggesting this PR not go through, or if it's a good step forward 🤔